### PR TITLE
w/ SB modified gds tune so it's cool with a frequency

### PR DESCRIPTION
**Pull request by jmfranck**

### CODE REVIEW

File: SpinCore_pp/gds_for_tune.py [hunk 1]
```diff
@@ -10,19 +10,16 @@
...
-if field is None: field = default_field
-if effective_gamma is None: effective_gamma = default_effective_gamma
-carrier_frequency = field*effective_gamma
+if len(sys.argv) == 1:
+    default_field = 3487.6 #3506.40 # G -- give this to 2 s.f.!!
+    print("Assuming a default field value of %f!! Probably not very wise!!"%default_field)
+    process_args(default_field)
+elif len(sys.argv) == 2:
+    carrier_frequency = process_args(sys.argv[1])
+elif len(sys.argv) == 3:
```
Comment (guinneak198) [inline SpinCore_pp/gds_for_tune.py:20]: so to use this we would type in the command prompt: "py gds_for_tune.py 3505.4" for setting a field of 3505.4 or "py gds_for_tune.py 14.889254" to set a carrier frequency of 14.889254 MHz? just double checking for clarity

**Review by guinneak198 — COMMENTED** [review]
AG good with changes for most part just question for clarity

File: SpinCore_pp/gds_for_tune.py (hunk 1 above)
Comment (jmfranck) [reply SpinCore_pp/gds_for_tune.py:20]: Exactly

**Review by aabeaton — COMMENTED** [review]
This looks ok to me

**Pull request merged**